### ISSUE TITLE
Crash when use an invalid font

### ISSUE THREAD

**Issue by leandrowilliam** [microsoft/terminal#1408]
# Environment

```none
Windows build number: 18362.175
Windows Terminal version (if applicable): 0.2.1715.0
```

# Steps to reproduce
Change settings an put `Cascadia Code` or another missing font

# Expected behavior
If the font is missing or wrong, keep Consoles by default, but don't crash
Maybe a hint or a message error, but don't crash

# Actual behavior
Crash and don't start until fix change the font

**Comment by DHowett-MSFT** [comment]
This is a direct duplicate of #550. Please make sure you search before filing a new issue.
